cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -1139,15 +1139,15 @@ def right_scheduler_schema(self, data, **kwargs):
     @validates_schema
     def same_subnet_in_different_queues(self, data, **kwargs):
         """Validate subnet_ids configured in different queues are the same."""
-        for queues in ["slurm_queues", "aws_batch_queues"]:
-            if queues in data:
+        queues = "slurm_queues"
```

Comment:
minor: we can avoid defining a variable for this.

Reply:
queues is used in  `if queues in data:` and `for q in data[queues] if _queue_has_subnet_ids(q)}`, So I define the variable here to remove duplication